js/livecoin.js

```diff
@@ -498,14 +516,17 @@ module.exports = class livecoin extends Exchange {
         if (typeof tag !== 'undefined')
             wallet += '::' + tag;
         let withdrawal = {
-            'amount': amount,
+            'amount': this.amountToPrecision (amount),
```

Comment:
The precision we need here is  `this.currencies[currency]['precision']`. Perhaps we could use decimalToPrecision instead?